project_role/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    project_role_ids = fields.One2many(
```

Comment:
This doesn't serve for multi-company roles, so solutions are:

1. Do this a compute field for only information purposes, and put a reference to where to remove this.
2. Remove the field at all.
3. Put a compute + inverse, feeding the other table property. My only fear with this is that you are unaware of the multi-company feature from this view, and then duplicate roles across companies.

Reply:
The original idea was to keep roles "owned" by companies, so this should be addressed if that would change

Reply:
Yes, that's why I'm proposing that solutions. I still fee that this is not important to be put here on the company form. At the end, normal project managers don't have access to this form, and this is not useful for system admins.